Frontend/src/components/collaboration-hub/CreatorMatchGrid.tsx

```
@@ -3,11 +3,12 @@ import CreatorMatchCard, { CreatorMatchCardProps } from "./CreatorMatchCard";
 
 interface CreatorMatchGridProps {
   creators: CreatorMatchCardProps[];
+  maxgridsize?: Number;
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)